android/src/main/java/com/reactnativecommunity/netinfo/ConnectivityReceiver.java

```diff
@@ -141,7 +141,7 @@ protected WritableMap createConnectivityEventMap(@Nullable final String requeste
         WritableMap event = Arguments.createMap();
 
         // Add if WiFi is ON or OFF
-        if (NetInfoUtils.isAccessWifiStatePermissionGranted(getReactContext())) {
+        if (NetInfoUtils.isAccessWifiStatePermissionGranted(getReactContext()) && mWifiManager != null) {
             boolean isEnabled = mWifiManager.isWifiEnabled();
             event.putBoolean("isWifiEnabled", isEnabled);
```

Comment:
This would be a breaking change, as the key would no longer be present, if we do it like this, then it will default to false, then if `mWifiManager` is non-null it will get the real state. What do you think?

```suggestion
        if (NetInfoUtils.isAccessWifiStatePermissionGranted(getReactContext())) {
            boolean isEnabled = false;
            if (mWifiManager != null) {
              isEnabled = mWifiManager.isWifiEnabled();
            }
            event.putBoolean("isWifiEnabled", isEnabled);
```

Reply:
Looking good 👍 Thank you!